service/src/main/java/org/cbioportal/service/impl/GeneServiceImpl.java

```diff
@@ -91,7 +100,7 @@ public Gene getGene(String geneId) throws GeneNotFoundException {
             throw new GeneNotFoundException(geneId);
         }
 
-        chromosomeCalculator.setChromosome(gene);
+        //chromosomeCalculator.setChromosome(gene);
```

Comment:
remove commented out line if not using anymore

Reply:
done.